change_management_customer_ref/__init__.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 # Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
```

Comment:
Hi @nikul-serpentcs  Those headers are also reserved for the authors or people who make significant contributions (e.g. adding new features). For migrations adding Serpent as contributor is enough in this case. Can you do the same in the other Pull Requests?

Reply:
@aheficent Sorry I was a bit confused in authors and Copyrights section, I thought we need to add ourself in Copyrights as we have contributed, but now it is clear, thanks!

I'll do the needful.